Sources/PointFree/Account/Invite.swift

```diff
@@ -118,43 +135,97 @@ let sendInviteMiddleware =
             parallel(sendInviteEmail(invite: invite, inviter: inviter).run)
               .run({ _ in })
 
-            return conn |> redirect(to: .account(.index))
+            return conn
+              |> redirect(
+                to: .account(.index),
+                headersMiddleware: flash(.notice, "Invite sent to \(invite.email.unwrap).")
+            )
           }
       }
 }
 
-private let showInviteView = View<(Database.TeamInvite, Database.User?)> { teamInvite, currentUser in
+let showInviteView = View<(Database.TeamInvite, Database.User, Database.User?)> { teamInvite, inviter, currentUser in
```

Comment:
Can `currentUser` ever be `.some` here?

Reply:
yeah it's possible that the user already has a PF account and is logged in when they click the accept invite link in their email